### ISSUE TITLE
Analyze "branchiness" of version trees.

### ISSUE THREAD

**Issue by bbengfort** [bbengfort/cloudscope#87]
Run a postprocessing script that computes the number of branches in the version tree for each object.

**Comment by bbengfort** [comment]
See #85

**Comment by bbengfort** [comment]
I've created a Tree data structure that can be loaded from versions to perform this analysis. I've also created an iPython notebook that explores several types of trees:
- Chain: every node has only one child except the last, which has zero.
- Prong: a small chain is forked at the end by two equal length branches.
- Trident: a small chain is forked at the end by two equal length branches and one smaller branch.
- Thorns: a chain where every other child has an additional child.
- Cactus: a chain where every other child has a branch with two nodes.
- BTree: a tree where every node has two children except the leaf nodes.
- Fan: the worst case scenario, no node has a child except the root.

To discuss their various properties with regards to the "branchiness" metric.
